catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/model/Segment.js

```diff
@@ -451,13 +466,18 @@ define([
             }
             //update and add values
             _.each(fields, function (field) {
-                field.saveData(backingData,ebrimTypes);
+                if (field.get('type') === 'date') {
+                    var dateToParse = (new Date(field.get('valueDate') + 'T' + field.get('valueTime') + 'Z')).toISOString().split('T');
```

Comment:
I would recommend looking at moment.js for this kind of parsing and splitting.

http://momentjs.com/docs/